src/Middleware/src/Headstart.Common/Assets/english-translations.json

```diff
@@ -497,7 +504,7 @@
     },
     "LOGIN": {
       "ACCOUNT_LOGIN": "Account Login",
-      "LOGIN": "Login",
+      "LOGIN": "Login with Anytime Fitness Account",
```

Comment:
It looks like some of these lines are reverted to SEB (headstart wouldn't want AF references).  Also not sure if we want HELPERMESSAGE on line 494.

Reply:
updated